packages/discovery-react-components/src/components/SearchResults/components/Result/__tests__/Result.test.tsx

```diff
@@ -3,53 +3,62 @@ import React from 'react';
 import { render, fireEvent, RenderResult } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import {
-  SearchContextIFC,
   searchResponseStoreDefaults,
   fetchDocumentsResponseStoreDefaults
 } from 'components/DiscoverySearch/DiscoverySearch';
-import DiscoveryV2 from 'ibm-watson/discovery/v2';
+import DiscoveryV2, { QueryResult } from 'ibm-watson/discovery/v2';
 
 import { wrapWithContext, browserWindow } from 'utils/testingUtils';
-import { SearchResults } from 'components/SearchResults/SearchResults';
+import { SearchResultsProps, SearchResults } from 'components/SearchResults/SearchResults';
+
+interface Setup {
+  searchResults: RenderResult;
+  selectResultMock: jest.Mock;
+}
+
+function setup(componentProps: Partial<SearchResultsProps>, QueryResults: QueryResult[]): Setup {
```

Comment:
It looks like there are a lot of times where `componentProps` is just `{}`. Can that be the second argument with a default so we only have to pass in a single argument for most of the `setup` uses?